include/seqan3/alphabet/container/concatenated_sequences.hpp

```diff
@@ -160,14 +57,18 @@ namespace seqan3
  * * Modifying elements is limited to operations on elements of that element, i.e. you can change a character,
  * but you can't assign a new member sequence to an existing position.
  *
+ * Note that the "value type" of seqan3::concatenated_sequences<T> is not `T`, it is a view––typically
+ * a std::span or a std::string_view. This view becomes invalid when the container is destroyed or any
+ * operation is performed on the container that invalidates its iterators, e.g. #push_back().
+ *
  * ### Example
  *
  * \include test/snippet/alphabet/container/concatenated_sequences.cpp
  *
  * ### Exceptions
  *
  * Whenever a strong exception guarantee is given for this class, it presumes that
- * `std::is_nothrow_move_constructible<typename inner_type::value_type>` otherwise only basic exception safety can
+ * `std::is_nothrow_move_constructible<typename underlying_container_type::value_type>` otherwise only basic exception safety can
  * be assumed.
```

Comment:
```suggestion
 * `std::is_nothrow_move_constructible<typename underlying_container_type::value_type>` otherwise only basic exception
 * safety can be assumed.
```